tests/auth_api/oidc/test_callbacks.py

```diff
@@ -129,7 +135,7 @@ def test__provide_invalid_state__should_return_status_400(
         ('internal_server_error', 'E0'),
         ('mitid_user_aborted', 'E1'),
     ))
-    def test__provide_oidc_errors__should_redirect_to_return_url_with_internal_error_code(
+    def test__provide_oidc_errors__should_redirect_to_return_url_with_internal_error_code(  # noqa 501
```

Comment:
Shall this apply only to this function, or in the whole file?

Reply:
Usually we only apply it to the line itself, to be on the safe side.